ansible/roles/csm.gpg_keys/tasks/main.yml

```diff
@@ -22,28 +22,14 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # Tasks for the csm.gpg_keys role
-- name: Fetch the HPE GPG Signing Key from the K8S secret
+- name: Fetch Public GPG Keys from the K8S secret
```

Comment:
This will fetch all keys, defined in k8s secret. How will keys get into k8s secret? CSM 1.5.x installer delivers a single key named `gpg-pubkey` into k8s secret. Support for multiple keys was added in 1.6.

Reply:
Isn't that this? https://github.com/Cray-HPE/csm/pull/3481/files

Reply:
This PR is for CSM 1.6.

You probably should add these comments to [the CSM 1.5 backport PR](https://github.com/Cray-HPE/csm-config/pull/279)